react-native-git-upgrade/cli.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+require('babel-register')({
+  presets: [
+    require('babel-preset-es2015-node'),
+    require('babel-preset-stage-3')
```

Comment:
The promise syntax was my first approach, https://github.com/ncuillery/react-native/commit/838b6c8121e15e29fe0f03cda6410933c35e1d7a#commitcomment-19170889

@bestander and me aren't huge fans of long promise chains...